### DISCUSSION TITLE
date of birth no longer populating automatically

### DISCUSSION THREAD

**Discussion by ms-adore** [sheltermanager/asm3#1485] [Q&A / Help]
Hello,

We have an online application form which uses the field name dateofbirth to pull the applicant's date of birth. This has auto-populated into the person record successfully for years, up until yesterday. In the Audit Trail, I now see  'DATEOFBIRTH': None whereas before we would see something like the following:   'DATEOFBIRTH': datetime.datetime(2003, 5, 22, 0, 0)

Did something change with this field?  Appreciate your help - its annoying to have to manually pull this over from the application.

**Comment by Tetzor** [comment]
Hi, we added new import fields for dateofbirthanimal and dateofbirthperson on the 3rd June, however dateofbirth should have continued to work in the same way. Thank you for letting us know about the problem, we have now deployed a fix.
